src/spec.ts

```diff
@@ -591,13 +590,12 @@ function normalizePathOverlay(spec: NormalizedUnitSpec, config: Config = {}): No
   }
 
   if (lineOverlay) {
-    const {interpolate} = markDef;
     layer.push({
       ...(projection ? {projection} : {}),
       mark: {
         type: 'line',
-        ...lineOverlay,
-        ...(interpolate ? {interpolate} : {})
+        ...pick(markDef, ['clip', 'interpolate']),
+        ...lineOverlay
```

Comment:
I think the order was wrong here before. @kanitw

Reply:
Ok -- another is don't you have to clip the point overlay below too?

Reply:
True, will change and merge.